webcompat/static/js/lib/wizard/helpers/postMessageParser.js

```diff
@@ -20,9 +18,6 @@ const handleMessage = (message) => {
 
   sendAnalyticsCS(utm_campaign, utm_source);
   if (url) {
-    showStep("category", {
-      url: extractPrettyUrl(url),
-    });
     updateUrl(url);
```

Comment:
I've also removed the "skip the url and scroll to the category step" when reporting through "Report site issue". It was part of the experiment to try and improve the form drop off. Thinking it might be confusing for new users as it happens immediately, skipping the part that we're looking for web compatibility issues (and "Learn more about web compatibility" pop up) .